Patterns/English/English-DateTime.yaml

```diff
@@ -530,8 +530,8 @@ AroundRegex: !simpleRegex
 AgoRegex: !simpleRegex
   def: \b(ago|before\s+(?<day>yesterday|today))\b
 LaterRegex: !nestedRegex
-  def: \b(?:later(?!(\s+in)?\s*{OneWordPeriodRegex})|from now|(from|after) (?<day>tomorrow|tmr|today))\b
-  references: [ OneWordPeriodRegex ]
+  def: \b(?:later(?!((\s+in)?\s*{OneWordPeriodRegex})|(\s+{TimeOfDayRegex}))|from now|(from|after) (?<day>tomorrow|tmr|today))\b
```

Comment:
I'm not sure this regex is the right place for this change.
Time of day has nothing to do with its purpose, which is to represent future times.

Reply:
Yes， but the issue is caused by "week later in the afternoon" was wrong extracted by "DateExtract". And this "laterRegex" is used in AgoLaterUtil, not used in cases like "later in this week", so we can see behind the "laterRegex"  can not be "OneWordPeriodRegex" which is "dateRange". I think also should limit "timeRange", so i add "timeOfDay" in the limit